whi_temperature_humidity:
  ros__parameters:
    loop_duration_temp: 4.0  # seconds
    loop_duration_noise: 3.0
    protocol_config: "/home/nvidia/ros2_ws/src/whi_temperature_humidity/config/serial_protocol.yaml"
    port: "/dev/ttyUART_485_2"
    baudrate: 9600

